templates/localgov/ce-dev/ansible/deploy.yml.j2

```diff
@@ -35,6 +35,10 @@
         no_dev: false
         working_dir: "{{ deploy_path }}"
         apcu_autoloader: false
+    - drush:
+        use_vendor: true
+    - drush_bin: "{{ deploy_path }}/vendor/drush/drush/drush"
```

Comment:
Is this better as `vendor/bin/drush`?

Reply:
without {{ deploy_path }} we have to play with '../../../vendor........'